.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove .vscode/settings.json file.  Feel free to exclude it from the repo via .gitignore.

Reply:
Done, i'll push changes when we wil be ok with the `/`